readme.html

```diff
@@ -1,29 +1,468 @@
+                <li><b>Examples:</b>
+                    <ul>
+                        <li>Create new rule with provider as
+                            "/orgs/1/sec_policy/draft/virtual_services/ce3387fd-703a-4068-a3d2-6e71d63068f4", consumer as
+                            "/orgs/1/sec_policy/active/ip_lists/958", ruleset as
+                            "/orgs/1/sec_policy/draft/rule_sets/1611", resolve_consumers_as "workloads", resolve_providers_as "virtual_services"
+                        </li>
+                        <ul>
+                            <li>consumer="/orgs/1/sec_policy/active/ip_lists/958"</li>
+                            <li>provider="/orgs/1/sec_policy/draft/virtual_services/ce3387fd-703a-4068-a3d2-6e71d63068f4"</li>
+                            <li>ruleset_href="/orgs/1/sec_policy/draft/rule_sets/1611"</li>
+                            <li>resolve_providers_as="virtual_services"</li>
+                            <li>resolve_consumers_as="workloads"</li>
+                        </ul>
+                    </ul>
+                </li>
+            </ul>
+        </li>
+        <li>
+            <h3>Create Enforcement Boundary</h3>
+            Creates an enforcement boundary with an ingress service using the given port/protocol.
+            <ul>
+                <li><b>Action Parameter: consumer</b></li>
+                <ul>
+                    <li>This parameter will accept href of consumers for creating new enforcement boundary.</li>
+                </ul>
+                <li><b>Action Parameter: provider</b></li>
+                <ul>
+                    <li>This parameter will accept href of providers for creating new enforcement boundary.</li>
+                </ul>
+                <li><b>Action Parameter: name</b></li>
+                <ul>
+                    <li>This parameter will accept name value for creating new enforcement boundary.</li>
+                </ul>
+                <li><b>Action Parameter: port</b></li>
+                <ul>
+                    <li>This parameter will accept the port value for a virtual service. It expects a numeric value as
+                        an input. An error message will be shown if the port value is invalid.</li>
+                </ul>
+                <li><b>Action Parameter: protocol</b></li>
+                <ul>
+                    <li>This parameter allows the user to select TCP or UDP from dropdown as protocol for virtual
+                        service</li>
+                </ul>
+                <li><b>Examples:</b>
+                    <ul>
+                        <li>Create new rule with provider as
+                            all workloads, consumer as
+                            "/orgs/1/sec_policy/active/ip_lists/958", name as "test-eb"
+                        </li>
+                        <ul>
+                            <li>consumer="/orgs/1/sec_policy/active/ip_lists/958"</li>
+                            <li>provider="ams"</li>
```

Comment:
Maybe add a note about what the AMS literal implies, or a link to Illumio documentation.

Reply:
added note